grafast/grafserv/src/servers/fastify/v4/index.ts

```diff
@@ -171,39 +176,48 @@ export class FastifyGrafserv extends GrafservBase {
     );
     */
 
-    const dynamicOptions = this.dynamicOptions;
-
-    app.route({
-      method:
-        this.dynamicOptions.graphqlOverGET ||
-        this.dynamicOptions.graphiqlOnGraphQLGET
-          ? ["GET", "POST"]
-          : ["POST"],
-      url: this.dynamicOptions.graphqlPath,
-      exposeHeadRoute: true,
-      bodyLimit: this.dynamicOptions.maxRequestLength,
-      handler: async (request, reply) => {
-        const digest = getDigest(request, reply);
-        const handlerResult = await this.graphqlHandler(
-          normalizeRequest(digest),
-          this.graphiqlHandler,
-        );
-        const result = await convertHandlerResultToResult(handlerResult);
-        return this.send(request, reply, result);
-      },
-      ...(this.resolvedPreset.grafserv?.websockets
-        ? {
-            wsHandler: makeHandler(makeGraphQLWSConfig(this)),
-          }
-        : null),
-    });
+    const {
+      graphiql,
+      graphiqlOnGraphQLGET,
+      graphiqlPath: url,
+      graphqlOverGET,
+      maxRequestLength: bodyLimit,
+      watch,
+    } = this.dynamicOptions;
+    const graphqlOverGet = (graphqlOverGET || graphiqlOnGraphQLGET) ?? false;
+    const websockets = this.resolvedPreset.grafserv?.websockets ?? false;
```

Comment:
Having `graphqlOverGet` and `graphqlOverGET` is very confusing!

```suggestion
    const websockets = this.resolvedPreset.grafserv?.websockets ?? false;
    const exposeGetRoute = graphqlOverGET || graphiqlOnGraphQLGET || websockets;
``````